### DISCUSSION TITLE
Please come back with the "Stretch Video" function, please.

### DISCUSSION THREAD

**Discussion by deXnnenor1** [redphx/better-xcloud#122] [General]
I used to use the ''Stretch Video'' function a lot, because, my monitor is not ''wide'', and it's not the best, please come back with the extension. Please.

**Comment by redphx** [comment]
What's your monitor's resolution/ratio?

**Comment by deXnnenor1** [comment]
1. My monitor model, from what I've ''researched'', is:  ''TGT-M153M60-01''. 
2. Resolution: 1024 × 768, 
3. ratio:  4:3 
4. Hope this helps you

**Reply by redphx** [reply]
Fixed in 1.15.1
